src/core/lib/url.jsx

```diff
@@ -40,4 +40,12 @@ function href({ gateway, widgetSrc, params }) {
   }
 }
 
-return { href };
+function getLinkUsingCurrentGateway(url) {
+  const data = fetch(`https://httpbin.org/headers`);
+  const gatewayURL = data?.body?.headers?.Origin ?? "";
+  return `https://${
+    gatewayURL.includes("near.org") ? "dev.near.org" : "near.social"
```

Comment:
We should change this to near.org instead of dev.near.org right?

Reply:
not necessarily because currently if we navigate to near.org, it force redirect to dev.near, but I will change to `near.org` so in future if dev.near doesn't work.